docs/postman/automated-test/BPDM Automated Test.postman_collection.json

```diff
@@ -0,0 +1,1112 @@
+{
+	"info": {
+		"_postman_id": "b0c17786-b741-4a06-a4b2-a0800981e2f9",
+		"name": "BPDM Automated Test",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "26818013"
+	},
+	"item": [
+		{
+			"name": "Create Metadata LE Identifier Type",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200 or 409\", function () {",
+							"    // Get the response status code",
+							"    var statusCode = pm.response.code;",
+							"",
+							"    // Check if the status code is either 200 or 409",
+							"    pm.expect(statusCode).to.be.oneOf([200, 409]);",
+							"});",
+							"pm.test(\"Body matches string\", function () {",
+							"    var responseBody = pm.response.text();",
+							"    var customLEId = pm.collectionVariables.get(\"customLEId\");",
+							"    pm.expect(responseBody).to.include(customLEId);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"var customLEId = \"CUSTOM_LE_ID_\" + Math.random().toString(36).substring(2, 12);",
```

Comment:
You can make your life easier by using the uuid functionality:

var uuid = require('uuid');
var customLEId =  uuid.v4();

Reply:
Updated with UUID, Thank you.